### DISCUSSION TITLE
Where is the mini-logo displayed?

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [kimai/kimai#2334] [Questions & Answers]
If I understand correctly, the mini-logo should be displayed at the top of the sidebar when it is closed.
but I don't see anything, only the first characters of the URL.
Does it have to have a specific size?

![Kimai2-MiniLogo](https://user-images.githubusercontent.com/20139133/107373975-63d7aa80-6ae7-11eb-940b-2629ae9339aa.png)

**Comment by kevinpapst** [comment]
The intention was to add a short text there. 

If you want to display an image, enter the HTML for an image:
`<img src="https://www.kimai.org/assets/icon/icon_32x32.png">`

**Reply by WhiteTiger-IT** [reply]
With <img src=""> tag the image is visible.
I had entered the URL, as in the field for the Logo.
Many tanks.
